plugin/vscode/performance-writer.js

```diff
@@ -0,0 +1,113 @@
+// Copyright (C) 2020  Matthew Glazar
+// See end of file for extended copyright information.
+
+const fs = require("fs");
+const path = require("path");
+const process = require("process");
+
+class PerformanceWriter {
+  constructor(extensionContext) {
+    if (extensionContext !== undefined) {
+      this._extensionContext = extensionContext;
+      this._tmpLogLines = [];
+      this._currentIndex = null;
+      this._fileIndex = 0;
+      this._writeToFileTimer = null;
+      this._timerActivated = false;
+      fs.mkdir(this._extensionContext.logUri.fsPath, (err) => {
+        if (!err) {
+          this.writeHeaderLine();
+        }
+      });
```

Comment:
Must fix: We aren't waiting for `mkdir` to finish before calling `createWriteStream`. I think this code is fragile and might not even work.